styleguide/templates/styleguide/index.html

```diff
@@ -338,6 +338,7 @@ <h4 class="heading-regular">Key-Value Details</h4>
 			If a new details table is needed for a specific page, create a new include
 			template file, following above structure
 		</p>
+	</details>
```

Comment:
separate from this, but noticed that the styleguide page was also missing a closing `</details>` tag.